<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<style>
body {font-family: Verdana;}

/* Style the tab */
.tab {
  overflow: hidden;
  border: 1px solid #ccc;
  background-color: #f1f1f1;
  align-items: center;
}

/* Style the buttons inside the tab */
.tab button {
  background-color: inherit;
  float: middle;
  border: none;
  outline: none;
  cursor: pointer;
  padding: 14px 16px;
  transition: 0.3s;
  font-size: 17px;
  text-align: center;
}

/* Change background color of buttons on hover */
.tab button:hover {
  background-color: rgb(225, 212, 212);
}

/* Create an active/current tablink class */
.tab button.active {
  background-color: #ccc;
}

/* Style the tab content */
.tabcontent {
  display: none;
  padding: 6px 12px;
  -webkit-animation: fadeEffect 1s;
  animation: fadeEffect 1s;
}

/* For image */
img.rounded {
  object-fit: cover;
  border-radius: 80%;
  height: 200px;
  width: 200px;
}

th, td {
  padding: 30px;
}

/* Fade in tabs */
@-webkit-keyframes fadeEffect {
  from {opacity: 0;}
  to {opacity: 1;}
}

@keyframes fadeEffect {
  from {opacity: 0;}
  to {opacity: 1;}
}

</style>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<title>Patrick Willems - Postdoctoral researcher</title>
</head>
<body bgcolor="#f1f1f1">

<div class="tab">
  <button class="tablinks" onclick="openTab(event, 'Patrick Willems')" id="defaultOpen">Patrick Willems</button>
  <button class="tablinks" onclick="openTab(event, 'Publications')">Publications</button>
  <button class="tablinks" onclick="openTab(event, 'Academic service')">Academic service</button>
  <button class="tablinks" onclick="openTab(event, 'CV')">CV</button>
</div>

<div id="Patrick Willems" class="tabcontent" style='margin-left:100px;margin-right:100px;'>
    <table style="width:100%">
        <tr style="width:100%">
            <th style="width:15%" valign="top" align="left" style="text-align: justify;">
                <img src="DSC_1646.JPG" class="rounded"><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Patrick Willems</b></font>
                <p align="justify"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;"><i>Postdoctoral researcher at Ghent University. Discovering biology guided by state-of-the-art OMICS technologies profiling from genome to metabolome.</i></p><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3"><i class="fa fa-map-marker" aria-hidden="true"></i> Ghent, Belgium<br>
                <span>&#9993;</span> <a href="mailto:willems533@gmail.com" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">willems533@gmail.com</a><br>
                <i class="fa">&#xf099;</i> <a href="https://twitter.com/Patrick14342109" style="text-decoration: none;" target="_blank" rel="noopener noreferrer"><font color="black">Patrick14342109</a><br>
                <img src="orcid.svg" height="14" width="14"> <a href="https://orcid.org/0000-0003-4667-2294" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">0000-0003-4667-2294</a><br>
                <img src="icons8-google-scholar-48.png" height="16" width="16"> <a href="https://scholar.google.com/citations?user=yh0bSnMAAAAJ&hl=en" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">Google scholar</a>
                <p align="bottom"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "2">© 2023 Patrick Willems.</p></font>
            </th>
            <th style="width:85%" valign="top" align="left">
                <p align="center"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Research experience</font><br><hr style="width=:90%; border: 0; height: 1px; background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));"></p>
                <p align="justify"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;">I obtained my master degree at Ghent University after plant studying cold stress signaling in plants during six months in the Vaughan Hurry lab in Umeå (Sweden). Afterwards, I continued studying plant abiotic stress signaling during my PhD in the Oxidative Stress Signaling group of Frank Van Breusegem in collaboration with the proteomics group of Kris Gevaert. My PhD work mostly entailed integrative –OMICS analysis of transcriptomic and proteomic datasets. As a post-doctoral researcher, I was active in the field of microbiology by annotating novel proteins in <i>Salmonella enterica</i> and infection biology. Afterwards, I re-oriented to study post-translational modifications (PTMs) in plants, with recent published work including experimental study of S-sulfenylation, AlphaFold2-based identification of disulfides and the development of an integrative plant PTM database ‘The Plant PTM Viewer’.</font></p>
            </th>
        </tr>
    </table>
</div>

<div id="Publications" class="tabcontent" style='margin-left:100px;margin-right:100px;'>
    <table style="width:100%">
        <tr style="width:100%">
            <th style="width:15%" valign="top" align="left" style="text-align: justify;">
                <img src="DSC_1646.JPG" class="rounded"><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Patrick Willems</b></font>
                <p align="justify"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;"><i>Postdoctoral researcher at Ghent University. Discovering biology guided by state-of-the-art OMICS technologies profiling from genome to metabolome.</i></p><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3"><i class="fa fa-map-marker" aria-hidden="true"></i> Ghent, Belgium<br>
                <span>&#9993;</span> <a href="mailto:willems533@gmail.com" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">willems533@gmail.com</a><br>
                <i class="fa">&#xf099;</i> <a href="https://twitter.com/Patrick14342109" style="text-decoration: none;" target="_blank" rel="noopener noreferrer"><font color="black">Patrick14342109</a><br>
                <img src="orcid.svg" height="14" width="14"> <a href="https://orcid.org/0000-0003-4667-2294" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">0000-0003-4667-2294</a><br>
                <img src="icons8-google-scholar-48.png" height="16" width="16"> <a href="https://scholar.google.com/citations?user=yh0bSnMAAAAJ&hl=en" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">Google scholar</a>
                <p align="bottom"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "2">© 2023 Patrick Willems.</p></font>
            </th>
            <th style="width:85%" valign="top" align="left">
                <p align="center"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Publications</font><br><hr style="width=:90%; border: 0; height: 1px; background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));"></p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;">For a full overview please visit my Google scholar or ORCID pages. Displayed below are corresponding and (shared) first authorships.<br>
                *Corresponding author, <sup><b>¶</b></sup>Shared first author
                <p align="justify">
                <i>Redox signaling in plants</i>
                <ul>
                    <li><b>Willems, Patrick<sup>¶</sup></b>, Valerie Van Ruyskensvelde<sup><b>¶</b></sup>, Takanori Maruta, Robin Pottie, Álvaro D. Fernández-Fernández, Jarne Pauwels, Matthew A. Hannah, Kris Gevaert, Frank Van Breusegem and Katrien Van der Kelen. 2023. “Mutation of Arabidopsis SME1 and Sm core assembly improves oxidative stress resilience.” Free Radical Biology and Medicine. Accepted.</li>
                    <li><b>Willems, Patrick</b>, Frank Van Breusegem, and Jingjing Huang. 2021. “Contemporary Proteomic Strategies for Cysteine Redoxome Profiling.” Plant physiology 186 (1): 110–124. doi:10.1093/plphys/kiaa074.</li>
                    <li>Wei, Bo<sup><b>¶</b></sup>, <b>Patrick Willems<sup>¶</sup></b>, Jingjing Huang, Caiping Tian, Jing Yang, Joris Messens, and Frank Van Breusegem. 2020. “Identification of Sulfenylated Cysteines in Arabidopsis Thaliana Proteins Using a Disulfide-Linked Peptide Reporter.” Frontiers in Plant Science 11. doi:10.3389/fpls.2020.00777.</li>
                    <li>Huang, Jingjing<sup><b>¶</b></sup>, <b>Patrick Willems<sup>¶</sup></b>, Bo Wei<sup><b>¶</b></sup>, Caiping Tian, Renan B Ferreira, Nandita Bodra, Santiago Agustín Martínez Gache, Khadija Wahni, Keke Liu, Didier Vertommen, Kris Gevaert, Kate S Carroll, Marc Van Montagu, Jing Yang, Frank Van Breusegem, and Joris Messens. 2019. “Mining for Protein S-Sulfenylation in Arabidopsis Uncovers Redox-Sensitive Sites.” PNAS 116 (42): 21256–21261. doi:10.1073/pnas.1906768116.</li>
                </ul>
                <i>Large-scale data analysis</i>
                <ul>
                    <li><b>Willems, Patrick*</b>, Jingjing Huang, Joris Messens, and Frank Van Breusegem. 2022. “Functionally annotating cysteine disulfides and metal binding sites in the plant kingdom using AlphaFold2 predicted structures.” Free Radical Biology and Medicine In press. doi: 10.1016/j.freeradbiomed.2022.12.001</li>
                    <li><b>Willems, Patrick</b>, Alison Horne, Thomas Van Parys, Sofie Goormachtig, Ive De Smet, Alexander Botzki, Frank Van Breusegem, and Kris Gevaert. 2019. “The Plant PTM Viewer, a Central Resource for Exploring Plant Protein Modifications.” The Plant Journal 99 (4): 752–762. doi:10.1111/tpj.14345.</li>
                    <li><b>Willems, Patrick</b>, Amna M’Hamdi, Simon Stael, Veronique Storme, Pavel Kerchev, Graham Noctor, Kris Gevaert, and Frank Van Breusegem. 2016. “The ROS Wheel: Refining ROS Transcriptional Footprints.” Plant Physiology 171 (3): 1720–1733.</li>
                </ul>  
                <i>Proteomics (DIA-MS, proteogenomics and N-terminal proteomics)</i>
                <ul>
                    <li><b>Willems, Patrick</b>, Elvis Ndah, Veronique Jonckheere, Frank Van Breusegem, and Petra Van Damme. 2021. “To new beginnings: Riboproteogenomics discovery of N-terminal proteoforms in Arabidopsis thaliana.” Frontiers in Plant Science. 12: 778804. doi:10.3389/fpls.2021.778804.</li>
                    <li><b>Willems, Patrick</b>, Ursula Fels, An Staes, Kris Gevaert, and Petra Van Damme. 2021. “Use of Hybrid Data-Dependent and -Independent Acquisition Spectral Libraries Empowers Dual-Proteome Profiling.” Journal of Proteome Research 20 (2): 1165–1177. doi:10.1021/acs.jproteome.0c00350.</li>
                    <li><b>Willems, Patrick</b>, Igor Fijalkowski, and Petra Van Damme. 2020. “Lost and Found : Re-Searching and Re-Scoring Proteomics Data Aids Genome Annotation and Improves Proteome Coverage.” mSystems 5 (5). doi:10.1128/mSystems.00833-20.</li>
                    <li><b>Willems, Patrick</b>, Elvis Ndah, Veronique Jonckheere, Simon Stael, Adriaan Sticker, Lennart Martens, Frank Van Breusegem, Kris Gevaert, and Petra Van Damme. 2017. “N-terminal Proteomics Assisted Profiling of the Unexplored Translation Initiation Landscape in Arabidopsis Thaliana.” Molecular & Cellular Proteomics 16 (6): 1064–1080.</li>
                </ul>
                </p>
            </th>
        </tr>
    </table>
</div>

<div id="Academic service" class="tabcontent" style='margin-left:100px;margin-right:100px;'>
    <table style="width:100%">
        <tr style="width:100%">
            <th style="width:15%" valign="top" align="left" style="text-align: justify;">
                <img src="DSC_1646.JPG" class="rounded"><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Patrick Willems</b></font>
                <p align="justify"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;"><i>Postdoctoral researcher at Ghent University. Discovering biology guided by state-of-the-art OMICS technologies profiling from genome to metabolome.</i></p><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3"><i class="fa fa-map-marker" aria-hidden="true"></i> Ghent, Belgium<br>
                <span>&#9993;</span> <a href="mailto:willems533@gmail.com" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">willems533@gmail.com</a><br>
                <i class="fa">&#xf099;</i> <a href="https://twitter.com/Patrick14342109" style="text-decoration: none;" target="_blank" rel="noopener noreferrer"><font color="black">Patrick14342109</a><br>
                <img src="orcid.svg" height="14" width="14"> <a href="https://orcid.org/0000-0003-4667-2294" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">0000-0003-4667-2294</a><br>
                <img src="icons8-google-scholar-48.png" height="16" width="16"> <a href="https://scholar.google.com/citations?user=yh0bSnMAAAAJ&hl=en" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">Google scholar</a>
                <p align="bottom"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "2">© 2023 Patrick Willems.</p></font>
            </th>
            <th style="width:85%" valign="top" align="left">
                <p align="center"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Academic service</font><br><hr style="width=:90%; border: 0; height: 1px; background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));"></p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;">Feel free to contact me for peer review, Ma/PhD juries, or other opportunities.<p>
                <i>Editing and peer review</i>
                <ul>
                    <li>Associate editor in Plant Proteomics section of Frontiers in Plants Science. Thus far, hosted one research topic "Plant protein termini: Their generation, modification and function" and handling editor of three articles.</li>
                    <li>Reviewer for international scientific journals including Cell Death & Differentiation, Journal of Proteome Research, Journal of Experimental Botany, Plant Physiology, Plant Physiology and Biochemistry, Frontiers in Plant Science.</li>
                </ul>
                <i>Student supervision</i>
                <ul>
                    <li>Supervision Muhammad Jawad SIDDIQUE, Ghent University, PhD. 2022-2023.</li>
                    <li>Co-supervision Lindsey De Veirman, Ghent University, Master 1 project, ‘Investigating thiol-based redox regulation using immunoprecipitation proteomics in plants’. 2022 March-April</li>
                    <li>Ricky Colman, Ghent University, Master 1 project, ‘Crosslink interactomics of the Mediator compex in Arabidopsis thaliana’. 2022 March-April.</li>
                    <li>Yanni Stalmans, HOWEST University of Applied Sciences, ‘Identifying and visualizing protein cleavage sites in Arabidopsis thaliana’. 2021 April-June, evaluation 18 June 2021.</li>
                </ul>
                <i>PhD/Ma jury member</i>
                <ul>
                    <li>October 2022: PhD jury member Bhavesh Parmar (KU Leuven), ‘Discovery of non-canonical translation products and functional annotation of a novel overlapping gene in C. elegans’’</li>
                    <li>June 2021: Reading commissioner, Master dissertation Valdes Snauwaert (Ghent University; WE10).</li>
                    <li>June 2022: Reading commissioner, Master dissertation Laure Simoens (Ghent University; WE10).</li>
                </ul>
                <i>Teaching experience</i>
                <ul>
                    <li>Aid Practical session, D012184A – Systems Biology, Ghent University.</li>
                    <li>Assist feedback evaluations thesis Master of Science in Plant Biotechnology</li>
                    <li>Participation Basisdocententraining 2.0 (NL), organized by the Department of Educational Policy – Educational Quality Assurance Office. </li>
                </ul>
            </th>
        </tr>
    </table>
</div>

<div id="CV" class="tabcontent" style='margin-left:100px;margin-right:100px;'>
    <table style="width:100%">
        <tr style="width:100%">
            <th style="width:15%" valign="top" align="left" style="text-align: justify;">
                <img src="DSC_1646.JPG" class="rounded"><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b>Patrick Willems</b></font>
                <p align="justify"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;"><i>Postdoctoral researcher at Ghent University. Discovering biology guided by state-of-the-art OMICS technologies profiling from genome to metabolome.</i></p><p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3"><i class="fa fa-map-marker" aria-hidden="true"></i> Ghent, Belgium<br>
                <span>&#9993;</span> <a href="mailto:willems533@gmail.com" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">willems533@gmail.com</a><br>
                <i class="fa">&#xf099;</i> <a href="https://twitter.com/Patrick14342109" style="text-decoration: none;" target="_blank" rel="noopener noreferrer"><font color="black">Patrick14342109</a><br>
                <img src="orcid.svg" height="14" width="14"> <a href="https://orcid.org/0000-0003-4667-2294" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">0000-0003-4667-2294</a><br>
                <img src="icons8-google-scholar-48.png" height="16" width="16"> <a href="https://scholar.google.com/citations?user=yh0bSnMAAAAJ&hl=en" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black">Google scholar</a>
                <p align="bottom"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "2">© 2023 Patrick Willems.</p></font>
            </th>
            <th style="width:85%" valign="top" align="left">
                <p align="center"><font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "4"><b><i>Curriculum vitae</i></font><br><hr style="width=:90%; border: 0; height: 1px; background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));"></p>
                <font face = "San Francisco,Roboto,Segoe UI,Helvetica Neue,Lucida Grande,Arial,sans-serif" size = "3" style="font-weight:normal;"></font>You can view my CV below or download <a href="https://github.com/patrick-willems/patrick-willems.github.io/raw/main/cv.pdf" target="_blank" rel="noopener noreferrer" style="text-decoration: none;"><font color="black"><u>here</u>. Please feel free to contact me for any questions.<p>
                <iframe src="https://github.com/patrick-willems/patrick-willems.github.io/raw/main/cv.pdf" width="100%" height="700px"></iframe>
                </p>
            </th>
        </tr>
    </table>
</div>

<script>
function openTab(evt, cityName) {
  var i, tabcontent, tablinks;
  tabcontent = document.getElementsByClassName("tabcontent");
  for (i = 0; i < tabcontent.length; i++) {
    tabcontent[i].style.display = "none";
  }
  tablinks = document.getElementsByClassName("tablinks");
  for (i = 0; i < tablinks.length; i++) {
    tablinks[i].className = tablinks[i].className.replace(" active", "");
  }
  document.getElementById(cityName).style.display = "block";
  evt.currentTarget.className += " active";
}

// Get the element with id="defaultOpen" and click on it
document.getElementById("defaultOpen").click();
</script>
   
</body>
</html> 
